### PR TITLE
Upgrade to Go 1.19

### PR DESCRIPTION
**Pull request by acald-creator**
Signed-off-by: Antonette Caldwell <pullmana8@gmail.com>

**Description**

This PR is related to https://github.com/meshery/meshery/issues/6220

**Notes for Reviewers**

- Partially enhanced workflows where appropriate

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ x ] Yes, I signed my commits.

### CODE REVIEW

File: install/docker-extension/Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM golang:1.17-alpine AS builder
+FROM golang:1.19-alpine AS builder
```
Comment (leecalcote) [inline install/docker-extension/Dockerfile:1]: Great job on finding these other buried Dockerfiles.

**Comment by leecalcote** [comment]
It might be nice to be able to explicitly override the golang version here - https://github.com/acald-creator/meshery/blob/104e73c19ee65f000f069b4ada020d4ae8d11415/.goreleaser.yml#L9-L10 - in the instance that a contributor needs to run `goreleaser` locally.

I ran into this need yesterday, but don't know that it's worth considering.

**Comment by Revolyssup** [comment]
@acald-creator Can you create a subsequent PR in meshery-extensions as well? We will merge both together.

**Comment by acald-creator** [comment]
@Revolyssup I will submit one sometime today.

**Comment by Chadha93** [comment]
@acald-creator It seems the `go.sum`  also need to be sync with the `go.mod `with this upgrade. Which should solve the golang workflow failure.

**Review by leecalcote — APPROVED** [review]
No small effort here, @acald-creator. 👩🏽‍💻

Some great work.

**Pull request merged**